lib/miq_expression.rb

```diff
@@ -1058,7 +1058,11 @@ def self.preprocess_managed_tag(tag)
   end
 
   def self.value2tag(tag, val = nil)
-    model, *values = tag.to_s.gsub(/[\.-]/, "/").split("/") # replace model path ".", column name "-" with "/"
+    if MiqExpression::Field.is_field?(tag)
+      model, *values = MiqExpression::Field.parse(tag).to_a
+    else
+      model, *values = tag.to_s.gsub(/[\.-]/, "/").split("/") # replace model path ".", column name "-" with "/"
+    end
 
     case values.first
```

Comment:
If this is only a tag thing, then it would be nice to have this logic embedded in `MiqExpression::Tag`